_virtual_machines/fedora-ws-35.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Fedora Workstation 35
+architecture: ARM64
+memory: 4 GiB
+disk: 5 GiB
+display: VGA
```

Comment:
The default display adapter is `virtio-ramfb` now

Reply:
I'll fix that.